Sources/PostgreSQL/Database/PostgreSQLDatabaseConfig.swift

```diff
@@ -32,47 +18,43 @@ public struct PostgreSQLDatabaseConfig {
     /// Optional password to use for authentication.
     public let password: String?
     
+    /// Which server to connect to.
+    public let serverAddress: PostgreSQLConnection.ServerAddress
+    
     /// Configures how data is transported to the server. Use this to enable SSL.
     /// See `PostgreSQLTransportConfig` for more info
-    public let transportConfig: PostgreSQLTransportConfig
+    public let transportConfig: PostgreSQLConnection.TransportConfig
+    
+    public init(hostname: String, port: Int = 5432, username: String, database: String? = nil, password: String? = nil, transport: PostgreSQLConnection.TransportConfig = .cleartext) {
+        self.init(serverAddress: .tcp(hostname: hostname, port: port), username: username, database: database, password: password, transport: transport)
+    }
     
-    /// Creates a new `PostgreSQLDatabaseConfig`.
-	public init(serverAddress: ServerAddress, username: String, database: String? = nil, password: String? = nil, transport: PostgreSQLTransportConfig = .cleartext) {
-		self.serverAddress = serverAddress
+	public init(serverAddress: PostgreSQLConnection.ServerAddress, username: String, database: String? = nil, password: String? = nil, transport: PostgreSQLConnection.TransportConfig = .cleartext) {
```

Comment:
A cosmetic thing but the indentation is wrong for this method, somehow tabs sneaked into a project indented by spaces.